### PR TITLE
doc: EKS Create Cluster Plugin

### PR DESCRIPTION
**Pull request by ashoknayak777**
_Copied from Old Repo_

### CODE REVIEW

**Review by abhibhaw — CHANGES_REQUESTED** [review]
`EnablePlugin ` why do we have a flag like enable plugin? Do I have any other functionality? to use this plugin in disable mode? I know these question can be answered by the person who created this plugin 😢

File: docs/user-guide/plugins/eks-create-cluster.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+e.g., `The EKS Create Cluster plugin is integrated for provisioning of EKS cluster.`
+
+### Input Variables
+
+| Variable                 | Format       | Description | Sample Value |
+| ------------------------ | ------------ | ----------- | ------------ |
+|   MinNodes               | STRING       | Minimum number of nodes in the EKS NodeGroup.          |      1        |
+|   MaxNodes               | STRING       | Maximum number of nodes in the EKS NodeGroup.          |      1        |
+|   UseEKSConfigFile       | BOOL         | Flag to use a config file for EKS cluster creation (true/false).    |      false        |
+|   EKSConfigFilePath      | STRING       | Path to the EKS cluster configuration file (required if `UseEKSConfigFile` flag is true).|      ~/.kube/config        |
```
Comment (abhibhaw) [inline docs/user-guide/plugins/eks-create-cluster.md]: Will this path be from my git repo? How will I supply the path? Also this example seems wrong, please confirm if that fits here

**Review by abhibhaw — CHANGES_REQUESTED** [review]

File: docs/user-guide/plugins/eks-create-cluster.md (hunk 1 above)
Comment (bhushan-nemade-dt) [reply docs/user-guide/plugins/eks-create-cluster.md]: fixed

**Review by abhibhaw — APPROVED** [review]
LGTM

**Review by ashoknayak777 — COMMENTED** [review]
LGTM

**Review by bhushan-nemade-dt — APPROVED** [review]

**Pull request merged**